apps/openchallenges/challenge-service/src/main/resources/openapi.yaml

```diff
@@ -682,6 +682,25 @@ components:
       format: date
       nullable: true
       type: string
+    EdamOperation:
```

Comment:
Nit: include `nullable: true`?

Reply:
See my reply to the other comment.

This file is auto-generated too.